src/router/index.js

```diff
@@ -51,7 +53,7 @@ const baseTitle = document.title
 Vue.use(Router)
 
 let mapsPath = generateUrl('/apps/maps')
-if (!isMapsInstalled) {
+if (!isMapsInstalled && isAppStoreEnabled) {
```

Comment:
Not sure that it makes sense to have that check here, the link will be hidden anyway, and the link will be wrong anyway too